src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
I agree that it's not a great user experience, but there must be a way to keep `/usr` off the RPP. Our tools all use the RPP as a search path. If it's unset the tools are all designed to return nothing since there is no path to search. In the case that RPP is set, it should always use exactly what's set. 

The debian guidelines say that it cannot need an environment so my proposed compromise is that an unset RPP defaults to use `/usr` if unset. The one case you called out is a little odd, but if you echoed RPP

```
$ echo $ROS_PACKAGE_PATH
$ export ROS_PACKAGE_PATH=$ROS_PACKAGE_PATH:`pwd`
$ echo $ROS_PACKAGE_PATH
:/path/to/pwd
```

It's very clear what your setting. There's no way for us to provide introspection to the default value because it's debian's requirement that it magically work without setting the value.

I would advocate for using the `RPP_NO_DEFAULTS` and that would turn off the "default" which is only used when the RPP is set. If we require setting `RPP_NO_DEFAULTS` even if you've explicitly set the RPP in your setup script we will break every single existing environment setup file and example setup file because suddenly their environment will now have unknown packages from `/usr` unless they set `RPP_NO_DEFAULTS`. Whereas if the default is only when RPP is unset, then the only people affected by the default value are the people not setting their RPP. This is the use case that the debian guidelines are targeted at. 

We just have to document that you cannot naively extend an unset path.